generator/src/OperatorTestGenerator.php

```diff
@@ -145,6 +147,8 @@ private function convertYamlTaggedValues(mixed $object): mixed
                 'bson_decimal128' => new Decimal128($value),
                 'bson_utcdatetime' => new UTCDateTime(is_numeric($value) ? $value : new DateTimeImmutable($value)),
                 'bson_binary' => new Binary(base64_decode($value)),
+                'bson_objectId' => new ObjectId($value),
+                'bson_uuid' => new Binary(hex2bin(str_replace('-', '', $value)), Binary::TYPE_UUID),
```

Comment:
Because I was curious, I went back and compared the original code you had with `pack('h*', ...)`. See below:

```php
<?php

// Initialized project with: composer require ramsey/uuid
require 'vendor/autoload.php';

use Ramsey\Uuid\Uuid;
use MongoDB\BSON\Binary;

$uuid = Uuid::uuid4();
var_dump($uuid);

$bytes = $uuid->getBytes();
$str = $uuid->toString();

$bin = new Binary($bytes, Binary::TYPE_UUID);
var_dump(bin2hex($bin));

$bin2 = new Binary(hex2bin(str_replace('-','',$str)), Binary::TYPE_UUID);
var_dump($bin == $bin2);

$bin3 = new Binary(pack('h*', str_replace('-','',$str)), Binary::TYPE_UUID);
var_dump($bin == $bin3);

$bin4 = new Binary(pack('H*', str_replace('-','',$str)), Binary::TYPE_UUID);
var_dump($bin == $bin4);
```

Output:

```
$ php foo.php 
class Ramsey\Uuid\Lazy\LazyUuidFromString#4 (2) {
  private ?Ramsey\Uuid\UuidInterface $unwrapped =>
  NULL
  private string $uuid =>
  string(36) "3a6224c2-f648-4978-b510-610c04fbbcd6"
}
string(32) "3a6224c2f6484978b510610c04fbbcd6"
bool(true)
bool(false)
bool(true)
```

So it looks like `h` (hex string, low nibble first) wouldn't have worked, but `H` (high nibble first) would have. `hex2bin()` is totally fine, though, and easier to read.